xml/docu_styleguide.smartdocs.xml

```diff
@@ -337,4 +337,58 @@ ultrashort description for social media, max. 55 chars&lt;/meta>
    </tgroup>
  </informaltable>
 </sect2>
+
+<sect2 xml:id="sec-revhistory">
+  <title>Revision history</title>
+  <para>Update revision history regularly, mentioning most important changes to
+  your document when you amend or rework it. The <quote>Revision History</quote> 
+  changelog is available as a link before the abstract and opens up in its individual
```

Comment:
We use "revision history" and "changelog" usually as interchangeable terms. However, we haven't really defined it so I would just avoid it (or move it as an additional term in the first sentence).

```suggestion
  your document when you amend or rework it. The data you enter as revision history
  is used as meta data. 
  The <quote>Revision History</quote> text is available as a link before the abstract and opens up in its individual
```